ios/browser/api/networking/models/BraveCertificateFingerprint.mm

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
```

Comment:
these definitely do not make sense for mm files inside brave/ios/browser. Maybe there's some header issue to be resolved for public headers in the framework, but the source files will always be built in brave-core